csrankings-a.csv

```diff
@@ -2175,6 +2175,7 @@ Avelino F. Zorzo,PUC-RS,http://www.inf.pucrs.br/~zorzo,rsiQlMgAAAAJ
 Avelino Francisco Zorzo,PUC-RS,http://www.inf.pucrs.br/~zorzo,rsiQlMgAAAAJ
 Avelino J. Gonzalez,University of Central Florida,http://www.cecs.ucf.edu/faculty/avelino-gonzalez,NOSCHOLARPAGE
 Avery Miller,University of Manitoba,http://www.cs.umanitoba.ca/~amiller,BvvdwvIAAAAJ
+Avesta Sasan,Univ. of California - Davis, https://www.ece.ucdavis.edu/~avesta/index.html,BMmfOCEAAAAJ

```

Comment:
Remove spurious spaces after commas. Please provide evidence for why faculty member meets inclusion criteria (ECE, not CS).